### PR TITLE
chore(docs): aliases for dotnet/chsarp docs.

### PR DESCRIPTION
**Pull request by avodovnik**
Starting to clean-up/add docs related to the C# port, beginning with aliases for `QuerySelector` functions.

### CODE REVIEW

**Review by aslushnikov — APPROVED** [review]

File: docs/src/api/class-elementhandle.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ ElementHandle instances can be used as an argument in [`method: Page.$eval`] and
 ## async method: ElementHandle.$
 * langs:
   - alias-python: query_selector
+  - alias-csharp: QuerySelectorAsync
```
Comment (pavelfeldman) [inline docs/src/api/class-elementhandle.md:62]: I think a better solution would be to consistently rename it to `querySelector` in all of our mds, then alias it into `$` only in JS

File: docs/src/api/class-elementhandle.md [hunk 2]
```diff
@@ -70,6 +71,7 @@ returns `null`.
 ## async method: ElementHandle.$$
 * langs:
   - alias-python: query_selector_all
+  - alias-csharp: QuerySelectorAllAsync
```
Comment (pavelfeldman) [inline docs/src/api/class-elementhandle.md:74]: ditto

File: docs/src/api/class-elementhandle.md (hunk 1 above)
Comment (avodovnik) [reply docs/src/api/class-elementhandle.md:62]: Going through that now, I *really* wish we'd use a language that would let me say "Find All References" :-)

**Comment by yury-s** [comment]
@avodovnik I rebased your patch on top of my changes, applied a couple more fixes and uploaded everything to https://github.com/yury-s/playwright/commits/avodovnik-dev/anvod/dotnet/docs (sorry, didn't manage to push to this PR directly, may be it's not possible), feel free to upload it here. We need to fix docs generator before committing this PR though as method names in js docs are not generated correctly (I tried with `SRC_DIR=~/playwright npm run roll`).

**Comment by yury-s** [comment]
Okay, docs generator is fixed in https://github.com/microsoft/playwright.dev/pull/97, this change is good to go.

**Comment by avodovnik** [comment]
@yury-s OK, I _think_ I got your changes from that branch over here, but I'd appreciate a quick cursory check.

File: docs/src/api/class-elementhandle.md [hunk 3]
```diff
@@ -112,18 +115,23 @@ assert tweet_handle.eval_on_selector(".like", "node => node.innerText") == "100"
 assert tweet_handle.eval_on_selector(".retweets", "node => node.innerText") = "10"
 ```
 
-### param: ElementHandle.$eval.selector = %%-query-selector-%%
+### param: ElementHandle.evalOnSelector.selector = %%-query-selector-%%
 
-### param: ElementHandle.$eval.expression = %%-evaluate-expression-%%
+### param: ElementHandle.evalOnSelector.pageFunction
```
Comment (yury-s) [inline docs/src/api/class-elementhandle.md:120]: this should be in javascript.md, looks like bad merge

File: docs/src/api/class-elementhandle.md [hunk 4]
```diff
@@ -159,11 +167,15 @@ feed_handle = page.query_selector(".feed")
 assert feed_handle.eval_on_selector_all(".tweet", "nodes => nodes.map(n => n.innerText)") == ["hello!", "hi!"]
 ```
 
-### param: ElementHandle.$$eval.selector = %%-query-selector-%%
+### param: ElementHandle.evalOnSelectorAll.selector = %%-query-selector-%%
+
+### param: ElementHandle.evalOnSelectorAll.pageFunction
```
Comment (yury-s) [inline docs/src/api/class-elementhandle.md:172]: this should be in javascript.md, looks like bad merge

**Pull request merged**